spec/features/inputs/slimselect_spec.rb

```diff
@@ -137,28 +132,28 @@
     end
 
     it "shows normal select control", js: true do
-      expect_select2_options_count_to_eq(0)
+      expect { expect_slimselect_options_count_to_eq(0) }.to raise_error(Capybara::ElementNotFound)
       expect(page).not_to have_selector("select.default-select")
-      expect(page).not_to have_selector("select.select2")
+      expect(page).not_to have_selector("select[data-id]")
     end
   end
 
-  context "when default config is Active Admin's default and select control has select 2 class" do
+  context "when default config is Active Admin's default and select control has slim-select class" do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [101/100]